### PR TITLE
Trim spaces from user given url

### PR DESCRIPTION
**Pull request by keltroth**
Forget the other PR, I made this one to avoid new commit on my own dev branch to come to you :)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
As I said, lgtm.

**Comment by Frenzie** [comment]
See #1777 for some (minimal) discussion. As I said in https://github.com/FreshRSS/FreshRSS/pull/1777#issuecomment-362551550

> Note that while trim does strip tabs (which you might accidentally get from tables) it doesn't strip non-breaking spaces (which you might also get from websites). I'm not sure if that matters. :-)

File: app/Controllers/feedController.php [hunk 1]
```diff
@@ -39,12 +39,14 @@ public function firstAction() {
 	 * @throws FreshRSS_Feed_Exception
 	 * @throws Minz_FileNotExistException
 	 */
-	public static function addFeed($url, $title = '', $cat_id = 0, $new_cat_name = '', $http_auth = '') {
+	public static function addFeed($param_url, $title = '', $cat_id = 0, $new_cat_name = '', $http_auth = '') {
```
Comment (Alkarex) [inline app/Controllers/feedController.php]: Please either keep `$url` (and later below use `$url = trim($url);`) or update the `@param` documentation a few lines higher up. I would better like option 1.

**Comment by Alkarex** [comment]
Thanks @keltroth 👍 
Please add your name to https://github.com/FreshRSS/FreshRSS/blob/dev/CREDITS.md

File: app/Controllers/feedController.php (hunk 1 above)
Comment (Frenzie) [reply app/Controllers/feedController.php]: I thought of writing something like that and decided against it without realizing that GitHub was hiding the documentation a few lines up. :-P

I'd also go with option 1.

Comment (keltroth) [reply app/Controllers/feedController.php]: Did option 1 👍

**Pull request merged**